package-lock.json

```diff
@@ -1378,6 +1378,12 @@
           "resolved": "https://registry.npmjs.org/async/-/async-0.9.2.tgz",
```

Comment:
Why was the package-lock.json regenerated? I don't see any changes to the package.json.

Reply:
It was re-generated when I ran npm install

Reply:
You might have regenerated it if you deleted it and ran npm i. It shouldn't be changing it just by running npm i.

Its no big deal but please don't delete package-lock.json unnecessarily.